### ISSUE TITLE
Missing some Lenovo Drivers for Windows 10 21H2

### ISSUE THREAD

**Issue by azulol** [maurice-daly/DriverAutomationTool#314]
Hey Guys,

since we want to change our sccm deployment to Win10 21H2, i want to upgrade our Driver Packages in SCCM.
Some drivers are missing, like L15, E15, P51s or T570.
I use the Driver Automation Tool Version 7.0.1.

Can someone help me or has the same problem?
Thank you.

**Comment by fredericlassalle** [comment]
Yes. I have the same problem. I think Lenovo must have changed the xml file.

**Comment by fredericlassalle** [comment]
I was in version 7.0.4. I switched back to version 6.5.6, it works.

Resolved by Jabbaxx
https://github.com/maurice-daly/DriverAutomationTool/issues/308
